utils.c

```diff
@@ -665,3 +665,13 @@ label_exec (const char *exec_label)
 #endif
   return 0;
 }
+
+int
+label_valid (const char *label)
+{
+#ifdef HAVE_SELINUX
+  if (is_selinux_enabled () > 0 && label)
+    return security_check_context ((security_context_t)label);
+#endif
+  return -1;
```

Comment:
I was only doing this to protect against future developers putting label_valid() call without checking label_support.

label_valid is not perfect, since it is not differentiating between file labels and process labels.  When bubblewrap eventually calls setexeccon() it will get validated then.